modules/core/keeper/msg_server.go

```diff
@@ -1131,6 +1131,18 @@ func (k Keeper) UpdateChannelParams(goCtx context.Context, msg *channeltypes.Msg
 }
 
 // PruneAcknowledgements defines a rpc handler method for MsgPruneAcknowledgements.
-func (Keeper) PruneAcknowledgements(goCtx context.Context, msg *channeltypes.MsgPruneAcknowledgements) (*channeltypes.MsgPruneAcknowledgementsResponse, error) {
+func (k Keeper) PruneAcknowledgements(goCtx context.Context, msg *channeltypes.MsgPruneAcknowledgements) (*channeltypes.MsgPruneAcknowledgementsResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	sequence, found := k.ChannelKeeper.GetAcknowledgementPruningSequence(ctx, msg.PortId, msg.ChannelId)
+	if !found {
+		return nil, errorsmod.Wrapf(channeltypes.ErrPruningSequenceNotFound, "port ID (%s) channel ID (%s)", msg.PortId, msg.ChannelId)
+	}
+
+	sequence = k.ChannelKeeper.PruneAcknowledgements(ctx, msg.PortId, msg.ChannelId, msg.Limit, sequence)
+
+	// update pruning sequence in store
+	k.ChannelKeeper.SetAcknowledgementPruningSequence(ctx, msg.PortId, msg.ChannelId, sequence)
+
 	return &channeltypes.MsgPruneAcknowledgementsResponse{}, nil
```

Comment:
Should we update the message to return the number of acks and receipts that were pruned?

Reply:
Let's do logging at least then.